configure.ac

```diff
@@ -176,9 +176,9 @@ AC_RUN_IFELSE(
 		case "$ssl_library_ver" in
 			101*)   ;; # 1.1.x
```

Comment:
Now that I think about it, if you want to have OpenSSL >= 3.1.x, you should remove the `101*` line, shouldn't you? That's ok to build against it (we're not using advanced features), but the comment doesn't match then.

Reply:
True, no point in going back to 1.x series either!